### PR TITLE
[DOCS] Rm vsix reference from docs; update Directory.Build.targets link

### PR DESCRIPTION
**Pull request by andrei-epure-sonarsource**
Follow-up of #5143

### CODE REVIEW

File: docs/contributing-analyzer.md [hunk 1]
```diff
@@ -99,26 +99,6 @@ If you run the project `ReviewDiffs` in debug mode, it will print in the output
 
 *PS*: This program outputs message in a way that is compatible with the [VsColorOutput](https://marketplace.visualstudio.com/items?itemName=MikeWard-AnnArbor.VSColorOutput) extension, so that errors are colored in red, and differences are colored like warnings.
 
-### Manual Tests
-
-From Visual Studio, make sure `SonarAnalyzer.Vsix.csproj` is selected as startup project. And then do the following:
```
Comment (andrei-epure-sonarsource) [inline docs/contributing-analyzer.md:104]: should we replace with some other instructions to debug analysis of a project (e.g. setting a `Debugger.Launch()`) ?

Comment (costin-zaharia-sonarsource) [reply docs/contributing-analyzer.md:104]: Yes, that would be great.

File: README.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ Standalone NuGet packages can be configured the same way as SonarLint in connect
 
 You can see how we do it in our own repository for SonarLint Connected mode (and you can reuse the same concept for the NuGet package):
 
-* We reference in each production-code project this [Directory.Build.props](analyzers/src/Directory.Build.props).
+* We reference in each production-code project this [Directory.Build.targets](analyzers/src/Directory.Build.targets).
```
Comment (andrei-epure-sonarsource) [inline README.md:116]: related to the https://github.com/SonarSource/sonar-dotnet/pull/5138 refactoring

**Review by costin-zaharia-sonarsource — APPROVED** [review]
LGTM! Thanks @andrei-epure-sonarsource, for taking care of this!

**Pull request merged**